### PR TITLE
exercise tic-tac-toe

### PR DESCRIPTION
**Pull request by paulotijero**
Nos falta el empate tecnico

### CODE REVIEW

File: week-1/the-pokes/tictactoe.rb [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  
+  value_human = []
+  value_bot = []
+
+  band = false
+
+  player = "X"
+  bot = "O"
+
+  table = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
```
Comment (DiegoCuevas) [inline week-1/the-pokes/tictactoe.rb]: puedes poner table = [" "] para crear un array vacio

File: week-1/the-pokes/tictactoe.rb [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        puts "El indescutible ganador!! es el BOT #{player}"
+        return band = true 
+      end
+      
+    end
+
+  end
+
+end
+
```
Comment (DiegoCuevas) [inline week-1/the-pokes/tictactoe.rb:108]: un mensaje diciendo que no se puede reingresar en un casillero ya tomado

File: week-1/the-pokes/tictactoe.rb [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+    # pos_human : Es el valor que ingresa el usuario
+    pos_human = turnHuman(notPlayed(value_human+value_bot))
+    table[pos_human-1] = player
+    value_human.push(pos_human)
+    
+
+    pos_bot = turnBot(notPlayed(value_human+value_bot))
+    table[pos_bot-1] = bot
```
Comment (DiegoCuevas) [inline week-1/the-pokes/tictactoe.rb:78]: "-" metodo indefinido

File: week-1/the-pokes/tictactoe.rb [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  player = "X"
+  bot = "O"
+
+  table = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+
+  while (band == false) do
+
+
+    # pos_human : Es el valor que ingresa el usuario
+    pos_human = turnHuman(notPlayed(value_human+value_bot))
```
Comment (sergiodxa) [inline week-1/the-pokes/tictactoe.rb]: I like how you composed those functions to create a result 👍

File: week-1/the-pokes/tictactoe.rb [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+    # player == "X" ? value_human.push(pos) : value_bot.push(pos)
+
+    puts "\n #{table[0]} | #{table[1]} | #{table[2]} "
+    puts "-----------"
+    puts " #{table[3]} | #{table[4]} | #{table[5]} "
+    puts "-----------"
+    puts " #{table[6]} | #{table[7]} | #{table[8]} "
+
+    puts "Soy un humna #{value_human}"
```
Comment (sergiodxa) [inline week-1/the-pokes/tictactoe.rb]: You have a typo here in `humna`
```suggestion
    puts "Soy un humano #{value_human}"
```

File: week-1/the-pokes/tictactoe.rb [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+        return band = true 
+      end
+      
+    end
+
+  end
+
+end
+
+puts tictactoe()
```
Comment (sergiodxa) [inline week-1/the-pokes/tictactoe.rb]: Why a `puts` here? You did `puts` inside `tictactoe` so I don't see a reason to run it here too, am I missing something? 🤔

File: week-1/the-pokes/tictactoe.rb (hunk 1 above)
Comment (paulotijero) [reply week-1/the-pokes/tictactoe.rb]: Maybe we can use this
`table = Array.new(8," ")`
-> This is used for print the table tic - tac - toe

File: week-1/the-pokes/tictactoe.rb (hunk 2 above)
Comment (paulotijero) [reply week-1/the-pokes/tictactoe.rb:108]: Nice advice!!

File: week-1/the-pokes/tictactoe.rb (hunk 3 above)
Comment (paulotijero) [reply week-1/the-pokes/tictactoe.rb:78]: We are going to check it

File: week-1/the-pokes/tictactoe.rb (hunk 6 above)
Comment (Gatts1) [reply week-1/the-pokes/tictactoe.rb]: you're right! We are going to take care next time :sweat_smile:

File: week-1/the-pokes/tictactoe.rb [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+  5,
+  6,
+  7,
+  8,
+  9
+]
+
+def notPlayed(value)
+  arrNP = TABLE - value
+  return arrNP
```
Comment (KattyaCuevas) [inline week-1/the-pokes/tictactoe.rb:37]: No es necesario crear una variable y usar el método `return`

File: week-1/the-pokes/tictactoe.rb (hunk 5 above)
Comment (MariethPerez) [reply week-1/the-pokes/tictactoe.rb]: Ok thanks,  we will correct.

File: week-1/the-pokes/tictactoe.rb [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+      puts "\nEs el turno del Humano"
+      value = gets.chomp.to_i
+      break if ( ( notplayed.include?(value)) && (value >= 1) && (value <=9) )
+    end
+    return value
+end
+
+def turnBot(value)
+  if(value != [])
+    value.sample
```
Comment (KattyaCuevas) [inline week-1/the-pokes/tictactoe.rb:52]: Se puede poner en una sola línea

File: week-1/the-pokes/tictactoe.rb [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+  1,
+  2,
+  3,
+  4,
+  5,
+  6,
+  7,
+  8,
+  9
+]
```
Comment (arturopuente) [inline week-1/the-pokes/tictactoe.rb]: Eso podría ser `TABLE = [*1..9]`

Comment (paulotijero) [reply week-1/the-pokes/tictactoe.rb]: Sure!

**Pull request merged**